### ISSUE TITLE
Google Analytics 4 support

### ISSUE THREAD

**Issue by clemp6r** [getredash/redash#6097]
### Issue Summary

It seems the Google Analytics integration is for version 3 only (aka Universal Analytics) and does not support version 4.

### Steps to Reproduce

1. Configure Google Analytics integration in Redash
2. Make a query by specifying a GA4 property id

Result: `Error running query: User does not have sufficient permissions for this profile.`

Unlike what is stated I'm pretty sure this is not a permission issue as I use the same service account for my GA3 property and the account has the same access level to all my properties.

### Technical details:

* Redash Version: 10.1.0
* How did you install Redash: AWS AMI + manual upgrades

**Comment by matthieubessol** [comment]
I run into the same problem and have set Analyze role for my redash account on GA4. Works well for Universal Analytics.

**Comment by cknoxrun** [comment]
Note there is a [PR](https://github.com/getredash/redash/pull/5868) active on this issue (not merged).

**Comment by justinclift** [comment]
It's merged now. :smile:

**Comment by JulioC** [comment]
@justinclift which version do I need to use this feature?

**Comment by justinclift** [comment]
Yeah, that's the critical question.  There's no "official" release with this in it yet, and we've paused the automated building of Docker images (that go onto Docker Hub) for now until we get our first release out the door.

You *could* build the image yourself (using this [developer setup guide](https://github.com/getredash/redash/wiki/Local-development-setup)), and then deploy it to a production oriented configuration (as created by the setup.sh script [here](https://github.com/getredash/setup/)).

That might take a bit of mucking around though to make work properly, as the recent development activity hasn't been focused on making nice, production stable builds (yet).

**Comment by Krikooo** [comment]
Hello @justinclift I tried to deploy "master" by building my own image and upgrading my previous instance of redash.
Once it runs, I, now, got some error with no css:
![image](https://github.com/getredash/redash/assets/129737743/0ada9c66-6a3c-4a77-b9f4-80786665c1e0)
```
Traceback (most recent call last):
  File "/app/redash/handlers/webpack.py", line 17, in get_asset
    with open(WEBPACK_MANIFEST_PATH) as fp:
FileNotFoundError: [Errno 2] No such file or directory: '/app/redash/handlers/../../client/dist/asset-manifest.json'
```
And login is not working :/

**Comment by justinclift** [comment]
Ahhh.  The lack of css means the front end pieces (html, javascript, etc) weren't included in the docker image that was built.

I've seen the same thing happen too, but haven't yet investigated that part.

I'll take a look at this in a few hours, as I've been meaning to get that figured out as well. :smile:

**Comment by Krikooo** [comment]
Thank you for your quick answer.
I went trought other issues and I understood that I forgot to add the build of the frontend in the docker image (by removing true for skip_frontend_build key).
Now I need to test GA4 👍

**Comment by justinclift** [comment]
Cool.  Yeah, that's probably the right path to take instead of waiting. :smile:

I'm getting really tired now, so might need to do the whole "investigate building docker image for this" tomorrow instead of today. :wink:

**Comment by Krikooo** [comment]
Hi, FYI, GA4 integration works like a charm 👌 
Thank you for your work!

**Comment by justinclift** [comment]
Awesome! :smile:

**Comment by roxanebellot** [comment]
Thank you for the issue and the comments :raised_hands: 

For anyone running into this issue, i allow myself to make some remarks:

- the "skip_frontend_build" key in docker-compose.yml must be `""`
- we put the images we built in [docker hub](https://hub.docker.com/repository/docker/octopusmindteam/redash-server/) if this can help somebody
